src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -22,7 +30,8 @@ function AppContent({ authorized, available, synced, children, isDataAvailable }
     <div className="disability-benefits-content">
       {available && authorized && canUseApp && !synced && <ClaimSyncWarning/>}
       {!authorized && <div className="row"><div className="columns medium-8"><ClaimsUnauthorized/></div></div>}
-      {authorized && (!available || !canUseApp) && unavailableView}
+      {authorized && !available && unavailableView}
+      {authorized && !canUseApp && unavailableMviRecords}
```

Comment:
We might want to check that this one is available, just to make sure we don't get duplicate messages:

`authorized && !canUseApp && available && unavailableMviRecords`